src/target/efm32.c

```diff
@@ -41,6 +41,7 @@
 #include "target_internal.h"
 #include "cortexm.h"
 #include "adiv5.h"
+#include <inttypes.h>
```

Comment:
Even though I included this, I still get compiler error. Any Idea why?

Reply:
Thanks @ALTracer for pointing out that PRIx64 is not supported in all embedded platforms. I opted to print unique as 2 u32 ints

Reply:
Technically `PRI*8` and `PRI*64` the C99 IO format specifiers are supported by Newlib, but BMF is restricted to newlib-nano on f103 etc. platforms, and that omits such support. I've found it out by rebuilding and linking BMF for blackpills with either newlib or nano and running it live for checking monitor output. Alternatively, examine `strings` resources in resulting .bin once you know which specifiers will be ignored by *printf at run time.